drivers/source/SFDP.cpp

```diff
@@ -377,20 +377,20 @@ int sfdp_detect_erase_types_inst_and_size(uint8_t *bptbl_ptr, sfdp_hdr_info &sfd
     return 0;
 }
 
-int sfdp_find_addr_region(bd_size_t offset, const sfdp_hdr_info &sfdp_info)
+int sfdp_find_addr_region(bd_addr_t offset, const sfdp_hdr_info &sfdp_info)
```

Comment:
Could you also update the declaration? https://github.com/ARMmbed/mbed-os/blob/798c3c5f4f7aa63210d90a43e4c1c920efcf69ed/drivers/include/drivers/internal/SFDP.h#L132

Reply:
Done